### PR TITLE
bench: add consensus breaking option to raise shred limits

### PR DESCRIPTION
**Pull request by ptaffet-jump**

### CODE REVIEW

File: src/app/fdctl/config/default.toml [hunk 1]
```diff
@@ -1021,9 +1021,24 @@ dynamic_port_range = "8900-9000"
         # second.  When benchmarking, this can be the limiting
         # bottleneck of the system, so this option is provided to raise
         # the limit.  If set to true, the limit will be lifted to
-        # 288,000,000 for a little over 1 million transactions a second.
+        # 288,000,000 for a little over 1 million cheap transactions a
```
Comment (mmcgee-jump) [inline src/app/fdctl/config/default.toml:1024]: Sorry my fault. Isn't this actually 480,000 TPS? (6*80,000) Can we update the number to be over 1M ?

File: src/app/fdctl/config/default.toml [hunk 2]
```diff
@@ -1021,9 +1021,24 @@ dynamic_port_range = "8900-9000"
...
+        # second.
         #
         # This option should not be used in production, and would cause
         # consensus violations and a consensus difference between the
         # validator and the rest of the network.
         larger_max_cost_per_block = false
+
+        # Solana has a consensus-agreed-upon limit of 32k data and
```
Comment (mmcgee-jump) [inline src/app/fdctl/config/default.toml:1032]: 32768? 32,000? 32KiB ?

**Review by mmcgee-jump — APPROVED** [review]

File: src/app/fdctl/config/default.toml (hunk 1 above)
Comment (ptaffet-jump) [reply src/app/fdctl/config/default.toml]: I updated it to 13x, which should be over 1M

File: src/app/fdctl/config/default.toml (hunk 2 above)
Comment (ptaffet-jump) [reply src/app/fdctl/config/default.toml]: Thanks, I clarified it's 32*1024

**Review by mmcgee-jump — APPROVED** [review]

**Pull request merged**